### PR TITLE
Add plugin version details in status endpoint

### PR DESCRIPTION
**Pull request by manishsubachchanyadav**
## Description

Added plugin version details for current and latest versions in the status endpoint.

## Screenshots

iScreen Shoter - Postman - 241113150547

@vplugins/websitepro

### CODE REVIEW

File: includes/Endpoints/Status.php [hunk 1]
```diff
@@ -87,5 +93,29 @@ public function get_status(WP_REST_Request $request) {
...
-    
+
+    /**
+     * Fetches the latest version of the plugin from the GitHub repository.
+     *
+     * @return string|null The latest version tag or null if unable to retrieve.
+     */
+    protected function fetch_latest_version_from_github() {
+        // GitHub URL for the specific repository
+        $url = "https://api.github.com/repos/vplugins/sm-post-connector/releases/latest";
```
Comment (rajanvijayan) [inline includes/Endpoints/Status.php]: Add this path in `includes/Helper/Globals.php` and render it from there.

**Review by rajanvijayan — CHANGES_REQUESTED** [review]

File: includes/Endpoints/Status.php (hunk 1 above)
Comment (manishsubachchanyadav) [reply includes/Endpoints/Status.php]: Sure @rajanvijayan

Comment (manishsubachchanyadav) [reply includes/Endpoints/Status.php]: @rajanvijayan Please review the changes; the path was already present in Globals.php and has now been rendered.

**Review by rajanvijayan — APPROVED** [review]
LGTM

**Pull request merged**